code/game/gamemodes/colonialmarines/halloween2016.dm

```diff
@@ -235,19 +235,19 @@
 	if(!num_marines)
 		if(mcguffin && mcguffin.loc)
 			round_finished = MODE_BATTLEFIELD_NT_MAJOR
-		else 
+		else
 			round_finished = MODE_BATTLEFIELD_NT_MINOR
 	else if(num_marines)
-		if(!mcguffin || !mcguffin.loc) 
+		if(!mcguffin || !mcguffin.loc)
```

Comment:
```suggestion
		if(!mcguffin?.loc)
```

Reply:
I didn't touch these lines at all, I'll be purging this entire thing in another PR.